source/get-started/create-a-connection-string.txt

```diff
@@ -0,0 +1,58 @@
+.. _ruby-get-started-connection-string:
+
+==========================
+Create a Connection String
+==========================
+
+You can connect to your MongoDB deployment by providing a
+**connection URI**, also called a *connection string*, which
+instructs the driver how to connect to a MongoDB deployment
+and how to behave while connected.
+
+The connection string includes the hostname or IP address and 
+port of your deployment, the authentication mechanism, user credentials 
+when applicable, and connection options.
+
+.. TODO:
+ To connect to an instance or deployment not hosted on Atlas, see
+ :ref:`ruby-connection-targets`.
+
+.. procedure::
+   :style: connected
+
+   .. step:: Find your MongoDB Atlas connection string
+
+      To retrieve your connection string for the deployment that
+      you created in the :ref:`previous step <ruby-get-started-create-deployment>`,
+      log into your Atlas account and navigate to the
+      :guilabel:`Clusters` section. Then, click the :guilabel:`Connect` button
+      for your new deployment.
+
+      .. figure:: /includes/figures/atlas_connection_connect_cluster.png
+         :alt: The connect button in the clusters section of the Atlas UI
+
+      Proceed to the :guilabel:`Connect your application` section. Select
+      "Ruby" from the :guilabel:`Driver` selection menu and the version
+      that best matches the version you installed from the :guilabel:`Version`
+      selection menu.
+
+   .. step:: Copy your connection string
+
+      Click the button on the right of the connection string to copy it to
+      your clipboard, as shown in the following screenshot:
```

Comment:
[s] For [accessibility purposes](https://www.mongodb.com/docs/meta/style-guide/writing/write-for-accessibility/#provide-context-for-ui-elements), maybe we can supplement the directional guidance with the name of the button. I noticed in the Atlas UI, this button is read by screenreaders as `Copy`.
```suggestion
      Click the :guilabel:`Copy` button on the right of the connection string to copy it to
      your clipboard, as shown in the following screenshot:
```

Reply:
Since the button isn't actually labeled with any text (just the copy symbol), I'm not sure if we should use :guilabel: - but I agree that just "the button on the right of the connection string" isn't ideal. I'll change to "the copy button"